components/display/ssd1306.rst

```diff
@@ -56,6 +56,7 @@ Configuration variables:
   - ``SH1106 128x64``
   - ``SH1106 96x16``
   - ``SH1106 64x48``
+  - ``SH1107 128x64``
```

Comment:
Want to add this to the SPI list below too?

Reply:
I went back and forth on this.  Based on how the ssd1306 component is written, theoretically the changes I have made to enable the use of a SH1107 over I2C also enabled the use over SPI, however [the screen](https://www.adafruit.com/product/4650) I'm testing with doesn't have the SPI interface populated, so the changes are untested.

I'm fine with including this but I'll leave this note here for future people letting them know that I wasn't able to test `SH1107 128x64` over SPI on real hardware.

Reply:
Thats fine, the screen driver (according to its datasheet) does support SPI so might as well just have it there just in case there is a module out there.